server/assets.py

```diff
@@ -41,21 +40,35 @@
     'https://cdnjs.cloudflare.com/ajax/libs/markdown-it/8.0.1/markdown-it.js',
     'https://cdn.rawgit.com/svbergerem/markdown-it-sanitizer/6efc1722/dist/markdown-it-sanitizer.js',
     'https://wzrd.in/standalone/markdown-it-anchor@2.6.0',
-    'https://cdnjs.cloudflare.com/ajax/libs/dropzone/4.3.0/dropzone.js',
     'https://cdn.ravenjs.com/2.1.0/raven.js',
     'js/main.js',
     filters='jsmin',
     output='public/js/common.js'
 )
 
+dropzone_css = Bundle(
+    'https://cdnjs.cloudflare.com/ajax/libs/dropzone/4.3.0/dropzone.css',
+    'css/dropzone.css',
+    filters='cssmin',
+    output='public/css/dropzone.css'
+)
+dropzone_js = Bundle(
+    'https://cdnjs.cloudflare.com/ajax/libs/dropzone/4.3.0/dropzone.js',
+    'js/dropzone.js',
+    filters='jsmin',
+    output='public/js/dropzone.js'
+)
+
 student_css = Bundle(
     'css/student.css',
+    'https://cdnjs.cloudflare.com/ajax/libs/pace/1.0.2/themes/blue/pace-theme-loading-bar.css',
```

Comment:
Included in student_css because now posting a comment also gets a loading bar now - usually those are processed quickly so I'm okay moving this to only be in `dropzone_css`